### ISSUE TITLE
Issue #709

### ISSUE THREAD

**Issue by HideakiAtsuyo** [libcpr/cpr#720]
## Description
It's been a long time I can't make a request anymore with cpr, I get Error 4 and no message and I only use a simple get request without any needed header etc...

## Expected Behavior
![](https://i.imgur.com/YSb7R4E.png)

## Actual Behavior
![](https://i.imgur.com/pJq8xOn.png)

## Steps to Reproduce
1. create a VS project with CPR and build/debug, example =>
![](https://i.imgur.com/PwGH3hT.png)

## Context
I can't get the response content.

## Your Environment
* Version used: 1.7.2
* Where did you get it from: vcpkg
* Operating System and version: Windows 10 (Build 19042)

Basically I have the same problem as #709

**Comment by COM8** [comment]
Thanks for this well done issue!
Since this is nothing we can fix I'm closing this issue in favour of #709.
So please either downgrade cpr from vcpkg to 1.6.x or wait for this PR to be merged: https://github.com/microsoft/vcpkg/pull/23984
